### DISCUSSION TITLE
Helm chart for new airflow version

### DISCUSSION THREAD

**Discussion by mikeslgoh** [apache/airflow#36317] [Q&A] [answered]
Hello,

I noticed that a new version of Airflow was released yesterday. I was wondering what the corresponding Helm chart would be for the new version.

Thanks

**Comment by Taragolis** [comment]
I hope you could find a good explanation in this discussion, because it is looks like simulator to your question:
- https://github.com/apache/airflow/discussions/35369

**Reply by mikeslgoh** [reply]
Just hoping to get confirmation on whether the existing helm chart will work with the new version or is a new helm chart going to be released since on the helm-charts repo, there is no mention of 2.8 yet

**Reply by Taragolis** [reply]
There is no breaking changes between minor version of Airflow, see: [Airflow’s release process and version policy](https://airflow.apache.org/docs/apache-airflow/stable/release-process.html#term-Feature-releases)


> helm-charts repo, there is no mention of 2.8 yet

From [Helm Chart for Apache Airflow](https://airflow.apache.org/docs/helm-chart/stable/index.html)

```
**Supported Airflow version: 1.10+, 2.0+**
```

**Reply by potiuk** [reply, marked as answer]
> Just hoping to get confirmation on whether the existing helm chart will work with the new version or is a new helm chart going to be released since on the helm-charts repo, there is no mention of 2.8 yet

There can't be mentioning of 2.8. There was no 2.8 available when the last helm chart was released. So it's impossible to mention it there from logic point of view.

Also if you rely on a software, I think it's a good idea to understand the release process for it. especially if it's documented. Also searching for existing issues and discussion before is a great idea (and it works nicely) that saves a ton of itme and distraction.

**Reply by potiuk** [reply]
Imagine if every user - like you - would require individual confirmation for their own particular version every time (instead of searching for documentation and answer that has already been given). It that would happen, then probably you would have to wait about a year for the next release because maintainers will be busy answering such questions.

It's of course exaggeration, but this is basically what you are asking for.

**Reply by mikeslgoh** [reply]
I had looked through open / closed issues and discussion threads and did not find questions related to helm charts or airflow versions (I had even checked in their helm chart repo). I personally never post questions unless I've already tried looking around and I thought this would be a fairly trivial question. Also what I meant by have it listed was just on their README here: https://github.com/airflow-helm/charts/tree/main/charts/airflow

**Reply by potiuk** [reply]
> Also what I meant by have it listed was just on their README here: https://github.com/airflow-helm/charts/tree/main/charts/airflow

Surely. because that is not the place you should look for - this is main. It will be updated when we prepare release notes and documetation, adding it in main have completely no effect what is in the released version, so whether 2.8 is there or not is irrelevant for the helm chart that has been released before 2.8 has been released.

**Reply by potiuk** [reply]
BTW. I hope this dicussion will be better indexed and will help others to find out the answers in the future.
